lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -125,6 +125,10 @@ for barclamp in $deactivation_list; do
     continue
   fi
 
+  if [[ $barclamp == "tempest" ]] && [[ $applied_proposals == Usage:* ]]; then
+    continue
+  fi
```

Comment:
This should be at the beginning of the list, shouldn't it?

And ideally, this would use the `case` syntax since we might have more optional barclamps at some point.